.github/workflows/build_and_test.yml

```diff
@@ -598,7 +598,7 @@ jobs:
           provenance: false
           push: true
           pull: true
-          file: Dockerfile.compute-node
+          file: compute/Dockerfile.compute-node
```

Comment:
NIT: as it's now in the subdir already, I'd just trim the `.compute-node`

Reply:
Renaming to `Containerfile` would also be great while we are at it.

Reply:
Huh, I had never heard of `Containtainerfile`s. TIL.

I'm going to stick to Dockerfile for now; that's the term we use everywhere else.

Reply:
Yep, Containerfile is the more generic name since it's no longer a docker specific file format.